packages/dynamic-import-vars/src/dynamic-import-to-glob.js

```diff
@@ -76,8 +76,24 @@ function shouldIgnore(glob) {
   return !containsAsterisk || containsIgnoredProtocol;
 }
 
-export function dynamicImportToGlob(node, sourceString) {
+const npmPackageNameAndGlobRegex = /^((@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*)\/(.+)$/;
+
+/**
+ *
+ * @param {unknown} node The AST node representing the dynamic import.
+ * @param {string} sourceString The fraction of the source code containing the dynamic import.
+ * @param {string} id The id of the module containing this dynamic import.
+ * @param {unknown} r The rollup instance
```

Comment:
Not a fan of abbreviations. Without the JSDoc, it would be close to impossible to figure out what it is supposed to mean (and it is also not an instance of Rollup). I would suggest `rollupContext` instead of `r`.